crypto/cipher_extra/cipher_extra.c

```diff
@@ -97,6 +97,15 @@ static const struct {
     {NID_rc4, "rc4", EVP_rc4},
 };
 
+static const struct {
+  const char* alias;
+  const char* name;
+} kCipherAliases[] = {
+    {"3des", "des-ede3-cbc"},
+    {"aes256", "aes-256-cbc"},
+    {"aes128", "aes-128-cbc"}
+};
```

Comment:
I thought about just making this mapping point to EVP_CIPHER to avoid the second lookup, but that could get out of date so I think this makes sense.